dev-python/cadquery-ocp/cadquery-ocp-7.5.3.0.ebuild

```diff
@@ -0,0 +1,195 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
```

Comment:
Is this ready for a bump to python3_10? It is the default now so it would be nice to add it as a PYTHON_COMPAT if possible.

Reply:
Changed to 10, will run the build overnight and see if anything breaks.

Reply:
build succeeded, the import test seems to pass